w4af/plugins/tests/grep/test_retirejs.py

```diff
@@ -86,6 +86,7 @@ def test_is_vulnerable_not_detected(self):
  - jQuery before 3.4.0, as used in Drupal, Backdrop CMS, and other products, mishandles jQuery.extend(true, {}, ...) because of Object.prototype pollution
  - Regex in its jQuery.htmlPrefilter sometimes may introduce XSS
  - Regex in its jQuery.htmlPrefilter sometimes may introduce XSS
+ - jQuery 1.x and 2.x are End-of-Life and no longer receiving security updates
```

Comment:
My pull request #623 hopefully addresses this issue, so you don't need to make this change. As @security-companion says, it's helpful if pull requests address one issue at a time (and not multiple issues in the same PR).